src/main/java/com/senkiv/bookstore/repository/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,41 @@
+package com.senkiv.bookstore.repository;
+
+import com.senkiv.bookstore.dto.BookSearchParametersDto;
+import com.senkiv.bookstore.model.Book;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    public static final String AUTHOR = "author";
+    public static final String ISBN = "isbn";
+    private static final String TITLE = "title";
```

Comment:
Make these constant public and reuse them in your providers. You will have one place for future changes